tests/meta/CMakeLists.txt

```diff
@@ -64,6 +64,7 @@ FOREACH(REFERENCE_FILE ${META_INTEGRATION_REFERENCES})
 	            AddMetaIntegrationTest(octave OctaveModular)
 	            AddMetaIntegrationTest(csharp CSharpModular)
 	            AddMetaIntegrationTest(ruby RubyModular)
+	            AddMetaIntegrationTest(scala ScalaModular)
```

Comment:
Finally, I changed my mind again, seeing how nicely travis executes the integration tests. So leave it in, it is redundant but I think its ok to have it